kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/security/JACCIdentityProvider.java

```diff
@@ -91,6 +92,31 @@ public String getName() {
 
     @Override
     public boolean hasRole(String s) {
+        Subject subject = getSubjectFromContainer();
+        if (subject != null) {
+            Set<Principal> principals = subject.getPrincipals();
+            if (principals != null) {
+                for (Principal principal : principals) {
+                    if (principal instanceof Group) {
+                        Enumeration<? extends Principal> groups = ((Group) principal).members();
+                        while (groups.hasMoreElements()) {
+                            Principal groupPrincipal = (Principal) groups.nextElement();
+                            if (groupPrincipal.getName().equals(s)) {
```

Comment:
Not really sure here, should we ignore case sensitive? What do you think?

Reply:
hmmm, good point, since there are not requirement about that, lets keep it this way, which is the default.